### DISCUSSION TITLE
Editing properties, Jan 2024

### DISCUSSION THREAD

**Discussion by gerontakos** [uwlib-cams/uwlswd_vocabs#34] [Announcements]
**This is a worksheet for editing the uwlswd property definitions, January, 2024** <br/>
* _The idea: write down the steps as a discussion (between Theo and David), create issues as needed; ultimately, close the discussion when all appropriate issues are created._
* Property to add: https://doi.org/10.6069/uwlib.55.d.3#appliesToMaterial
* Pertinent resources:
    *  UWLSWD properties namespace: https://doi.org/10.6069/uwlib.55.d.3#
    * [Minimal description of a property for UWLSWD](https://github.com/uwlib-cams/uwlswd/wiki/01_resource_guidelines#2-minimal-description-of-a-property)
        * Edit this description, if appropriate
        * Follow.when applicable, what is in the minimal description for concepts and concept schemes -- see below for some changes needed
* Changes needed to make a correct minimal-description-of-a-property:
    * Correct:
        * Several bullets are misaligned 
        * description of how the rdfs:comment is used
    * Added (did David add the following?):
        *  rdfs:isDefinedBy = https://doi.org/10.6069/uwlib.55.d.3
        *  rdfs:label = $label
    * Add:
        * either skos:historyNote or dct:description  
* The description of the properties dataset as a whole looks good, except:
    * update version; should be done by main.py; just check after running the script that it becomes 1-5-1 
    * manually remove 
        * void:classPartition triple
        * remove all void:feature triples
* Add a issue stating we may need a minimal description of a dataset, possibly to [01_resource_guidelines](https://github.com/uwlib-cams/uwlswd/wiki/01_resource_guidelines); that's a question for the planners
    * what will be the rdf:type? void:Dataset?

### Needed to get started on anything:
* XML editor
* Local Saxon processor (that's needed, right?)
* Python and some dependencies
* Java
* A preferred way to display Python; Notepad++? Launch Anaconda? Visual Studio Code?

**Comment by briesenberg07** [comment]
Hello @gerontakos ! I just came across this, luckily, hadn't received any notification. Is this something you are looking for feedback on? CC @cspayne

**Comment by gerontakos** [comment]
No, this ok, you can ignore this for now, apologies. David and I just scrawled some notes then took a few weeks off. We return to the work on Friday.
